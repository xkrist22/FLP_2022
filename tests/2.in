E,T,F
a,m,l,r,i
E
E->EaT
E->T
T->TmF
T->F
F->lEr
F->i

