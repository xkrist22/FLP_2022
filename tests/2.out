E,T,F
a,m,l,r,i
E
E->EaT
E->TmF
E->lEr
E->i
T->TmF
T->lEr
T->i
F->lEr
F->i

